test/email_reply_parser_test.rb

```diff
@@ -209,7 +209,7 @@ def test_mulitple_on
   def test_pathological_emails
     t0 = Time.now
     email("pathological")
-    assert (Time.now - t0) < 1, "Took too long, upgrade to re2 gem."
+    assert (Time.now - t0) < 1, "Took too long, upgrade to re2 gem. See https://rubygems.org/gems/re2"
```

Comment:
Slightly tweaking the error message language:

```suggestion
    assert (Time.now - t0) < 1, "Parsing took too long, please install the re2 gem locally. See https://rubygems.org/gems/re2"
```

Reply:
Ah, maintainers aren't able to edit the upstream. 🙂Accepting as-is. :+1: